flutter/lib/benchmark/benchmark.dart

```diff
@@ -233,6 +233,33 @@ class BenchmarkState extends ChangeNotifier {
         datasetsError.map((element) => '\n${++index}) $element').join();
   }
 
+  Future<String> validateOfflineMode(String errorDescription) async {
+    if (!_store.offlineMode) {
+      return '';
+    }
+    final errors = <String>[];
+    for (final job in _getBenchmarkJobs()) {
+      final modelPath = job.benchmark.benchmarkSetting.src;
```

Comment:
Do we really want to check model path?
It is defined by the backend, and there is nothing we can do to change it except to edit sources and rebuild the app.
I though that this offline mode check was about making sure that we use the appropriate `tasks` file, while using the release binary.

Reply:
From my understanding that some other engineers may build the app on his own and use his models to test.

@Mostelk Can you clarify?